Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
category_idx: 122
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
Category index: 122
Categories: <QuerySet [<Category: Guild>, <Category: viseguild>, <Category: speacker>]>
Category index: 122
Categories: <QuerySet [<Category: Guild>, <Category: viseguild>, <Category: speacker>]>
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
Response: CON Welcome to UR voting
1. Continue as voter
2. Exit
Response: CON Enter Registration Number:

Response: CON Enter your PIN:

Response: CON Choose category to vote:
1. Guild
2. viseguild
3. speacker
4. Back
Category index: 122
Categories: <QuerySet [<Category: Guild>, <Category: viseguild>, <Category: speacker>]>
Response: END Invalid category selection.
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /urvoting/
Traceback (most recent call last):
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
  File "/Users/apple/philz_devs/python/voting_app/vot/home/views.py", line 191, in ussdapp
    voter = Voters.objects.get(student__regNumber=registration_number)
UnboundLocalError: local variable 'registration_number' referenced before assignment
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /urvoting/
Traceback (most recent call last):
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
  File "/Users/apple/philz_devs/python/voting_app/vot/home/views.py", line 190, in ussdapp
    voter = Voters.objects.get(student__regNumber=registration_number)
UnboundLocalError: local variable 'registration_number' referenced before assignment
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /urvoting/
Traceback (most recent call last):
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
  File "/Users/apple/philz_devs/python/voting_app/vot/home/views.py", line 192, in ussdapp
    voter = Voters.objects.get(student__regNumber=registration_number)
UnboundLocalError: local variable 'registration_number' referenced before assignment
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /urvoting/
Traceback (most recent call last):
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
  File "/Users/apple/philz_devs/python/voting_app/vot/home/views.py", line 191, in ussdapp
    voter = Voters.objects.get(student__regNumber=registration_number)
UnboundLocalError: local variable 'registration_number' referenced before assignment
Internal Server Error: /urvoting/
Traceback (most recent call last):
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
  File "/Users/apple/philz_devs/python/voting_app/vot/home/views.py", line 191, in ussdapp
    voter = Voters.objects.get(student__regNumber=registration_number)
UnboundLocalError: local variable 'registration_number' referenced before assignment
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /urvoting/
Traceback (most recent call last):
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
  File "/Users/apple/philz_devs/python/voting_app/vot/home/views.py", line 144, in ussdapp
    voter = Voters.objects.get(student__regNumber=registration_number)
UnboundLocalError: local variable 'registration_number' referenced before assignment
Internal Server Error: /urvoting/
Traceback (most recent call last):
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
  File "/Users/apple/philz_devs/python/voting_app/vot/home/views.py", line 144, in ussdapp
    voter = Voters.objects.get(student__regNumber=registration_number)
UnboundLocalError: local variable 'registration_number' referenced before assignment
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /urvoting/
Traceback (most recent call last):
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
  File "/Users/apple/philz_devs/python/voting_app/vot/home/views.py", line 191, in ussdapp
    voter = Voters.objects.get(student__regNumber=registration_number)
UnboundLocalError: local variable 'registration_number' referenced before assignment
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /urvoting/
Traceback (most recent call last):
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
  File "/Users/apple/philz_devs/python/voting_app/vot/home/views.py", line 163, in ussdapp
    response += f"99*{registration_number}"
UnboundLocalError: local variable 'registration_number' referenced before assignment
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /urvoting/
Traceback (most recent call last):
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/fields/__init__.py", line 2688, in to_python
    return uuid.UUID(**{input_form: value})
  File "/usr/local/Cellar/python@3.9/3.9.10/Frameworks/Python.framework/Versions/3.9/lib/python3.9/uuid.py", line 177, in __init__
    raise ValueError('badly formed hexadecimal UUID string')
ValueError: badly formed hexadecimal UUID string

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
  File "/Users/apple/philz_devs/python/voting_app/vot/home/views.py", line 186, in ussdapp
    category = Category.objects.get(categoryId=category_id)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/query.py", line 623, in get
    clone = self._chain() if self.query.combinator else self.filter(*args, **kwargs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/query.py", line 1436, in filter
    return self._filter_or_exclude(False, args, kwargs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/query.py", line 1454, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/query.py", line 1461, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/sql/query.py", line 1545, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/sql/query.py", line 1576, in _add_q
    child_clause, needed_inner = self.build_filter(
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/sql/query.py", line 1491, in build_filter
    condition = self.build_lookup(lookups, col, value)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/sql/query.py", line 1318, in build_lookup
    lookup = lookup_class(lhs, rhs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/lookups.py", line 27, in __init__
    self.rhs = self.get_prep_lookup()
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/lookups.py", line 341, in get_prep_lookup
    return super().get_prep_lookup()
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/lookups.py", line 85, in get_prep_lookup
    return self.lhs.output_field.get_prep_value(self.rhs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/fields/__init__.py", line 2672, in get_prep_value
    return self.to_python(value)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/fields/__init__.py", line 2690, in to_python
    raise exceptions.ValidationError(
django.core.exceptions.ValidationError: ['“1” is not a valid UUID.']
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /urvoting/
Traceback (most recent call last):
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
  File "/Users/apple/philz_devs/python/voting_app/vot/home/views.py", line 163, in ussdapp
    response += f"99*{registration_number}"
UnboundLocalError: local variable 'registration_number' referenced before assignment
Internal Server Error: /urvoting/
Traceback (most recent call last):
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
  File "/Users/apple/philz_devs/python/voting_app/vot/home/views.py", line 163, in ussdapp
    response += f"99*{registration_number}"
UnboundLocalError: local variable 'registration_number' referenced before assignment
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /urvoting/
Traceback (most recent call last):
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/fields/__init__.py", line 2688, in to_python
    return uuid.UUID(**{input_form: value})
  File "/usr/local/Cellar/python@3.9/3.9.10/Frameworks/Python.framework/Versions/3.9/lib/python3.9/uuid.py", line 177, in __init__
    raise ValueError('badly formed hexadecimal UUID string')
ValueError: badly formed hexadecimal UUID string

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
  File "/Users/apple/philz_devs/python/voting_app/vot/home/views.py", line 186, in ussdapp
    category = Category.objects.get(categoryId=category_id)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/query.py", line 623, in get
    clone = self._chain() if self.query.combinator else self.filter(*args, **kwargs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/query.py", line 1436, in filter
    return self._filter_or_exclude(False, args, kwargs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/query.py", line 1454, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/query.py", line 1461, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/sql/query.py", line 1545, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/sql/query.py", line 1576, in _add_q
    child_clause, needed_inner = self.build_filter(
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/sql/query.py", line 1491, in build_filter
    condition = self.build_lookup(lookups, col, value)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/sql/query.py", line 1318, in build_lookup
    lookup = lookup_class(lhs, rhs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/lookups.py", line 27, in __init__
    self.rhs = self.get_prep_lookup()
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/lookups.py", line 341, in get_prep_lookup
    return super().get_prep_lookup()
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/lookups.py", line 85, in get_prep_lookup
    return self.lhs.output_field.get_prep_value(self.rhs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/fields/__init__.py", line 2672, in get_prep_value
    return self.to_python(value)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/fields/__init__.py", line 2690, in to_python
    raise exceptions.ValidationError(
django.core.exceptions.ValidationError: ['“1” is not a valid UUID.']
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /urvoting/
Traceback (most recent call last):
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/fields/__init__.py", line 2688, in to_python
    return uuid.UUID(**{input_form: value})
  File "/usr/local/Cellar/python@3.9/3.9.10/Frameworks/Python.framework/Versions/3.9/lib/python3.9/uuid.py", line 177, in __init__
    raise ValueError('badly formed hexadecimal UUID string')
ValueError: badly formed hexadecimal UUID string

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
  File "/Users/apple/philz_devs/python/voting_app/vot/home/views.py", line 186, in ussdapp
    category = Category.objects.get(categoryId=category_id)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/query.py", line 623, in get
    clone = self._chain() if self.query.combinator else self.filter(*args, **kwargs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/query.py", line 1436, in filter
    return self._filter_or_exclude(False, args, kwargs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/query.py", line 1454, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/query.py", line 1461, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/sql/query.py", line 1545, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/sql/query.py", line 1576, in _add_q
    child_clause, needed_inner = self.build_filter(
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/sql/query.py", line 1491, in build_filter
    condition = self.build_lookup(lookups, col, value)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/sql/query.py", line 1318, in build_lookup
    lookup = lookup_class(lhs, rhs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/lookups.py", line 27, in __init__
    self.rhs = self.get_prep_lookup()
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/lookups.py", line 341, in get_prep_lookup
    return super().get_prep_lookup()
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/lookups.py", line 85, in get_prep_lookup
    return self.lhs.output_field.get_prep_value(self.rhs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/fields/__init__.py", line 2672, in get_prep_value
    return self.to_python(value)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/fields/__init__.py", line 2690, in to_python
    raise exceptions.ValidationError(
django.core.exceptions.ValidationError: ['“1” is not a valid UUID.']
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /urvoting/
Traceback (most recent call last):
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/fields/__init__.py", line 2688, in to_python
    return uuid.UUID(**{input_form: value})
  File "/usr/local/Cellar/python@3.9/3.9.10/Frameworks/Python.framework/Versions/3.9/lib/python3.9/uuid.py", line 177, in __init__
    raise ValueError('badly formed hexadecimal UUID string')
ValueError: badly formed hexadecimal UUID string

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
  File "/Users/apple/philz_devs/python/voting_app/vot/home/views.py", line 186, in ussdapp
    category = Category.objects.get(categoryId=category_id)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/query.py", line 623, in get
    clone = self._chain() if self.query.combinator else self.filter(*args, **kwargs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/query.py", line 1436, in filter
    return self._filter_or_exclude(False, args, kwargs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/query.py", line 1454, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/query.py", line 1461, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/sql/query.py", line 1545, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/sql/query.py", line 1576, in _add_q
    child_clause, needed_inner = self.build_filter(
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/sql/query.py", line 1491, in build_filter
    condition = self.build_lookup(lookups, col, value)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/sql/query.py", line 1318, in build_lookup
    lookup = lookup_class(lhs, rhs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/lookups.py", line 27, in __init__
    self.rhs = self.get_prep_lookup()
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/lookups.py", line 341, in get_prep_lookup
    return super().get_prep_lookup()
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/lookups.py", line 85, in get_prep_lookup
    return self.lhs.output_field.get_prep_value(self.rhs)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/fields/__init__.py", line 2672, in get_prep_value
    return self.to_python(value)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/db/models/fields/__init__.py", line 2690, in to_python
    raise exceptions.ValidationError(
django.core.exceptions.ValidationError: ['“1” is not a valid UUID.']
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /urvoting/
Traceback (most recent call last):
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/utils/deprecation.py", line 136, in __call__
    response = self.process_response(request, response)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/contrib/sessions/middleware.py", line 59, in process_response
    request.session.save()
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/contrib/sessions/backends/db.py", line 80, in save
    return self.create()
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/contrib/sessions/backends/db.py", line 54, in create
    self.save(must_create=True)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/contrib/sessions/backends/db.py", line 82, in save
    obj = self.create_model_instance(data)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/contrib/sessions/backends/db.py", line 69, in create_model_instance
    session_data=self.encode(data),
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/contrib/sessions/backends/base.py", line 94, in encode
    return signing.dumps(
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/core/signing.py", line 152, in dumps
    return TimestampSigner(key=key, salt=salt).sign_object(
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/core/signing.py", line 250, in sign_object
    data = serializer().dumps(obj)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/core/signing.py", line 127, in dumps
    return json.dumps(obj, separators=(",", ":")).encode("latin-1")
  File "/usr/local/Cellar/python@3.9/3.9.10/Frameworks/Python.framework/Versions/3.9/lib/python3.9/json/__init__.py", line 234, in dumps
    return cls(
  File "/usr/local/Cellar/python@3.9/3.9.10/Frameworks/Python.framework/Versions/3.9/lib/python3.9/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/local/Cellar/python@3.9/3.9.10/Frameworks/Python.framework/Versions/3.9/lib/python3.9/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/usr/local/Cellar/python@3.9/3.9.10/Frameworks/Python.framework/Versions/3.9/lib/python3.9/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type UUID is not JSON serializable
Internal Server Error: /urvoting/
Traceback (most recent call last):
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/utils/deprecation.py", line 136, in __call__
    response = self.process_response(request, response)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/contrib/sessions/middleware.py", line 59, in process_response
    request.session.save()
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/contrib/sessions/backends/db.py", line 80, in save
    return self.create()
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/contrib/sessions/backends/db.py", line 54, in create
    self.save(must_create=True)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/contrib/sessions/backends/db.py", line 82, in save
    obj = self.create_model_instance(data)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/contrib/sessions/backends/db.py", line 69, in create_model_instance
    session_data=self.encode(data),
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/contrib/sessions/backends/base.py", line 94, in encode
    return signing.dumps(
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/core/signing.py", line 152, in dumps
    return TimestampSigner(key=key, salt=salt).sign_object(
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/core/signing.py", line 250, in sign_object
    data = serializer().dumps(obj)
  File "/Users/apple/philz_devs/python/voting_app/env_vot/lib/python3.9/site-packages/django/core/signing.py", line 127, in dumps
    return json.dumps(obj, separators=(",", ":")).encode("latin-1")
  File "/usr/local/Cellar/python@3.9/3.9.10/Frameworks/Python.framework/Versions/3.9/lib/python3.9/json/__init__.py", line 234, in dumps
    return cls(
  File "/usr/local/Cellar/python@3.9/3.9.10/Frameworks/Python.framework/Versions/3.9/lib/python3.9/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/local/Cellar/python@3.9/3.9.10/Frameworks/Python.framework/Versions/3.9/lib/python3.9/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/usr/local/Cellar/python@3.9/3.9.10/Frameworks/Python.framework/Versions/3.9/lib/python3.9/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type UUID is not JSON serializable
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
/Users/apple/philz_devs/python/voting_app/vot/home/views.py changed, reloading.
Watching for file changes with StatReloader
